ava-test/which.js

```diff
@@ -0,0 +1,47 @@
+import fs from 'fs';
+
+import test from 'ava';
+
+import shell from '..';
+
+shell.config.silent = true;
+
+//
+// Invalids
+//
+
+test('no args', t => {
+  shell.which();
+  t.truthy(shell.error());
+});
+
+test('command does not exist in the path', t => {
+  const result = shell.which('asdfasdfasdfasdfasdf'); // what are the odds...
+  t.falsy(shell.error());
+  t.falsy(result);
+});
+
+//
+// Valids
+//
+
+// TODO(nate): make sure this does not have a false negative if 'git' is missing
+test('basic usage', t => {
+  const git = shell.which('git');
```

Comment:
Node can be installed as `nodejs` on Linux. It's always installed as `node.exe` on Windows, though. We can count on git to be always be installed on a dev's box. That's the rationale. We could probably change the next one to `git` as well if we wanted to be consistent